doc/sheet.md

```diff
@@ -3,78 +3,21 @@
 Mística provides a sheet component that can be used to display a modal-like content from over the main content
 of the screen.
 
-## Basic usage
+## Predefined sheets
 
-You can show any content you want inside the sheet by passing it as a child of the component.
-
-```jsx
-import {Sheet} from 'mistica';
-
-const MyComponent = () => {
-  const [showSheet, setShowSheet] = useState(false);
-  return (
-    <>
-      <ButtonPrimary onPress={() => setShowSheet(true)}>show sheet</ButtonPrimary>
-      {showSheet && (
-        <Sheet onClose={() => setShowSheet(false)}>
-          <Placeholder />
-        </Sheet>
-      )}
-    </>
-  );
-};
-```
-
-The sheet will close when the user does the swipe down gesture or when the background overlay is touched. The
-`onClose` callback is called when the closing animation finishes, that's the right place to unmount the sheet
-as shown in the example above.
-
-You can also close the sheet programmatically using the render prop:
-
-```jsx
-import {Sheet} from 'mistica';
-
-const MyComponent = () => {
-  const [showSheet, setShowSheet] = useState(false);
-  return (
-    <>
-      <ButtonPrimary onPress={() => setShowSheet(true)}>show sheet</ButtonPrimary>
-      {showSheet && (
-        <Sheet onClose={() => setShowSheet(false)}>
-          {({closeModal, modalTitleId}) => (
-            <>
-              <Title1 id={modalTitleId}>My sheet</Title1>
-              <Placeholder />
-              <ButtonPrimary onPress={closeModal}>Close</ButtonPrimary>
-            </>
-          )}
-        </Sheet>
-      )}
-    </>
-  );
-};
-```
-
-## Sheet with predefined content
-
-Mística predefines some common sheet patterns for you to use: `RadioListSheet`, `ActionsListSheet`,
-`InfoSheet` and `ActionsSheet`. You can see examples in the storybook.
-
-## `showSheet` imperative api
+Some predefined sheets are available: `RadioListSheet`, `ActionsListSheet`, `InfoSheet` and `ActionsSheet`.
+You can see examples in Storybook.
 
-Instead of using React components, there is an alternative way to show a sheet: using the `showSheet`
-function. For this to work, you need to render a `<SheetRoot/>` somewhere in your app, typically where you
-render the mistica `<ThemeContextProvider/>`, but it could be anywhere.
+To use them, first you must configure the `SheetRoot` component in your app:
 
 ```jsx
 import {SheetRoot} from '@telefonica/mistica';
 
 export const App = () => {
   return (
-    <>
-      <SheetRoot />
-      <RestOfYourApp />
-    </>
+    <SheetRoot>
+      <MyApplication />
+    </SheetRoot>
```

Comment:
SheetRoot doesn't create a context, it's a singleton, so this shouldn't be needed

Reply:
It can be used both ways, but  I find more "react-ish" using it wrapping the children.